restapi.test/restapi_SimpleUnitTest.cs

```diff
@@ -0,0 +1,30 @@
+using Microsoft.AspNetCore.Mvc;
+using NUnit.Framework;
+using Sepes.restapi.ValuesController; //Add the controller to this test
+using System.Linq; //Adds some mathematics
+
+
+namespace Tests
+{
+
+    public class SimpleUnitTest
+    {
+        [SetUp]
+        public void Setup()
+        {
+        }
+
+        [Test]
+        public void AddNumbers()
+        {
+            int[] testProducts = {2,3};
+            var controller = new ValuesController(); //Make the controller object
+            ActionResult<int> result = controller.AddNumbers(testProducts[0], testProducts[1]); //Call funtion and pass in variables
```

Comment:
Yep, array was leftover from the debugging and attempts to emulate microsoft example.
It was left inside with the idea it can be used to make test accept dynamic input for testing of edge values. But I agree its unecesarry for this as no production logic exists